src/django/api/views/facility/facilities_view_set.py

```
@@ -844,128 +839,83 @@ def claim(self, request, pk=None):
+
             return Response({
                 'pending': pending,
                 'approved': approved,
             })
-        except Facility.DoesNotExist as exc:
-            raise NotFound() from exc
-        except Contributor.DoesNotExist as exc:
-            raise NotFound() from exc
+
+        except (Facility.DoesNotExist, Contributor.DoesNotExist):
+            raise NotFound()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)